src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityAnalysisRunResult.cs

```diff
@@ -9,22 +9,39 @@ public class SensitivityAnalysisRunResult
    {
       private readonly List<PKParameterSensitivity> _allPKParameterSensitivities = new List<PKParameterSensitivity>();
 
+      /// <summary>
+      ///    This property won't be serialized
+      /// </summary>
+      private readonly List<OutputParameterSensitivity> _allOutputParameterSensitivities = new List<OutputParameterSensitivity>();
```

Comment:
We don't save this in the DB for now...if we want to ever display it, then we can think about it